pkg/datapath/loader/netlink.go

```diff
@@ -93,8 +94,12 @@ func replaceDatapath(ctx context.Context, ifName, objPath, progSec, progDirectio
 			"obj", objPath, "sec", progSec}
 	} else {
 		loaderProg = "tc"
+		var tcprio string
```

Comment:
```suggestion
		var tcPrio string
```
Nitpick: Would be easier to read with a bit `P`, it would avoid reading `TCP`-something.